clusterctl/README.md

```diff
@@ -8,22 +8,30 @@ Read the [experience doc here](https://docs.google.com/document/d/1-sYb3EdkRga49
 
 ### Prerequisites
 
-Follow the steps listed at [CONTRIBUTING.md](https://github.com/kubernetes-sigs/cluster-api/blob/master/cluster-api/clusterctl/CONTRIBUTING.md) to:
-
-1. Build the `clusterctl` tool
-
-```
-go build
+1. Install [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) 
+2. Build the `clusterctl` tool
+
+```bash
+$ cd $GOPATH/src/sigs.k8s.io/
+$ git clone https://github.com/<GITHUB_USERNAME>/cluster-api.git
+$ cd $GOPATH/src/sigs.k8s.io/cluster-api/clusterctl/
+$ go build
 ```
  
-2. Create a `machines.yaml` file configured for your cluster. See the provided template for an example.
-
 ### Limitations
 
 
 ### Creating a cluster
 
-**NOT YET SUPPORTED!**
+1. Create a `cluster.yaml` and `machines.yaml` files configured for your cluster. See the provider specific templates and generation tools at `$GOPATH/src/sigs.k8s.io/cluster-api/clusterctl/examples/<provider>`. 
```

Comment:
It would be nice to add an example of using the generate-yaml script here. Do you expect that to be a common pattern for each cloud provider?

Reply:
I am not sure. I added some more documentation in the examples folder. We can upstream to the clusterctl README.md if it becomes a reality that the generation is common across providers.